commcare_connect/microplanning/filters.py

```
@@ -12,6 +14,89 @@
+        method="_noop",
+        widget=forms.DateInput(attrs={"type": "date", "class": INPUT_CSS}),
+    )
+    end = django_filters.DateFilter(
+        label=_("To"),
+        method="_noop",
+        widget=forms.DateInput(attrs={"type": "date", "class": INPUT_CSS}),
+    )
+
+    class Meta:
+        model = WorkArea
+        fields = []
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)